app/forms/bulk_update/bulk_add_trainees_upload_form.rb

```diff
@@ -41,6 +38,15 @@ def csv
 
   private
 
+    def create_upload
+      BulkUpdate::TraineeUpload.create!(
+        provider: provider,
+        file: file,
+        number_of_trainees: csv&.count,
+        status: valid? ? :pending : :failed,
```

Comment:
Is this going to be ever `:failed` since we check at [23](https://github.com/DFE-Digital/register-trainee-teachers/pull/4734/files#diff-2e324da006cb703e645d47853fb95bcb14e51ee36f696dd9513e782525ea1eaeR23) if the form object is `valid?` before we create the record?